### DISCUSSION TITLE
WAS Node Suite IMPORT FAILED

### DISCUSSION THREAD

**Discussion by anuplohade** [WASasquatch/was-node-suite-comfyui#245] [Q&A] [answered]
WAS suite is failing to import into comfy ui suddenly and it seems CSV file has something to do with it.
CSV file format is correct and path is also added into 'was_suite_config.json'
Python Version: 3.10.6

Could anyone please help in resolving this issue?
(Please check screenshots for reference.)
![Screenshot 2023-10-23 225212](https://github.com/WASasquatch/was-node-suite-comfyui/assets/72828505/5f41533a-afbe-4527-b2f2-02fe6294f2f2)
![Screenshot 2023-10-23 225302](https://github.com/WASasquatch/was-node-suite-comfyui/assets/72828505/c70acf5e-7d9e-49a4-a82e-7984750a886e)
![Screenshot 2023-10-23 225352](https://github.com/WASasquatch/was-node-suite-comfyui/assets/72828505/98d86930-88ad-462f-b6fd-c90cde0ebc7a)
![Screenshot 2023-10-23 225425](https://github.com/WASasquatch/was-node-suite-comfyui/assets/72828505/dbb3cc7e-1f91-4786-9767-aeeae33a5ef1)

**Comment by WASasquatch** [comment]
And csv is in expected format, for example, containing "name" for the style?

The was extras error can be resolved by updating it and cstr

**Comment by anuplohade** [comment, marked as answer]
I was able to resolve it.

I had to change the line 309 in WAS_Node_Suite.py and rename it to the column heading of style from CSV file i.e. 'style_name'

Secondly I changed CSV file to UTF-8 encoding format.

**Reply by dkamajix** [reply]
Im having was node import problems myself, can you please be more clear as to what exactly Id have to change in the WAS_Node_Suite.py, I don't really understand what I need to change here. my line 309 may be different? If you can be specific Id really appreciate
![image](https://github.com/user-attachments/assets/e22cf6cb-7b4e-484f-b13b-87e7e23665da)
